build/windows/build_windows.py

```diff
@@ -124,6 +144,9 @@ def copy_dll(dest):
   '-DCMAKE_SYSTEM_VERSION=' + SYSTEM_VERSION
 ]
 
+if CXX_STANDARD != '':
+    cmake_cmd.extend(['-DCMAKE_CXX_STANDARD=' + CXX_STANDARD])
+
```

Comment:
better move this to build_common.py to support this on all platforms

Reply:
https://github.com/heroiclabs/nakama-cpp/pull/52/commits/434d691af72804bd015d3887d1ed9734c3ee7b50